conversation/ShareJS
# Remove unused features

Rather than maintain a bunch of stuff that we don't and never plan to use in production (for our purposes at least), removing what we don't use should lighten the load and make it easier to upgrade some of the very out of date packages.

The diff is "large", though you will quickly see the bulk of the changes are just removing code, there's not really any way around this.

The summary of the changes is:
 - Ace client and examples removed
 - Etherpad client and examples removed
 - Redis, couchdb, mysql, amazon (dynamodb), mongo storage options removed
 - A bunch of examples either removed or changed to use CodeMirror
 
How to test:

1) Run `./bin/exampleserver`
2) Navigate to http://localhost:8000/demos.html
3) Check that all the remaining demos function
